docs/en/setup/backend/backend-alarm.md

```diff
@@ -180,6 +180,23 @@ slackHooks:
     - https://hooks.slack.com/services/x/y/z
 ```
 
+## WeChat Hook
+To use Wechat webhook you need to follow the [Wechat Webhooks guide](https://work.weixin.qq.com/help?doc_id=13376).
```

Comment:
```suggestion
Note, only WeCom(WeChat Company Edition) supports webhook. To use the WeChat webhook you need to follow the [Wechat Webhooks guide](https://work.weixin.qq.com/help?doc_id=13376).
```

Reply:
`WeCom` is from the official website.

![image](https://user-images.githubusercontent.com/5441976/92988725-b2652680-f500-11ea-883f-a569770bb20a.png)

Reply:
changed